src/easier68k/core/enum/op_size.py

```diff
@@ -0,0 +1,52 @@
+"""
+Op Size Enum
+Holds the values for various binary translations of size codes
+"""
+
+class BigSSize:
```

Comment:
I think we should try to find the naming used by the 68k manual instead

This is specified on page 220, maybe call it MoveSize since it's only used by MOVE.

Reply:
Good idea, I don't like these names either, hahaha.

Reply:
Looking at your next comments, MoveSize seems fine, and then SmallSize and SingleBitSize.